src/storage/src/storage/read_object.rs

```diff
@@ -528,6 +528,31 @@ impl ReadObjectResponse {
                     .set_or_clear_crc32c(headers_to_crc32c(headers))
                     .set_md5_hash(headers_to_md5_hash(headers))
             }),
+            storage_class: headers
+                .get("x-goog-storage-class")
+                .and_then(|sc| sc.to_str().ok())
+                .map(|sc| sc.to_string())
+                .unwrap_or_default(),
```

Comment:
nit: this repeats the same expression 5 times, maybe a helper function that consumes the header name?

Reply:
Added a helper.